src/components/general/CoverPhoto.tsx

```diff
@@ -17,29 +17,49 @@ const messages = {
   imageName: 'Cover Photo'
 }
 
-const CoverPhoto = props => {
+type CoverPhotoProps = {
+  userId: number | null
+  coverPhotoSizes: CoverPhotoSizes | null
```

Comment:
lil nit: we have a nullable utility type we've been using for this stuff: `Nullable<number>`